ui/app/AppLayouts/Wallet/HistoryTab.qml

```diff
@@ -1,8 +1,140 @@
 import QtQuick 2.3
+import "../../../imports"
 
 Item {
-    Text {
-        id: name3
-        text: "HISTORY"
+    Component {
+      id: transactionListItem
+
+      Item {
+        anchors.right: parent.right
+        anchors.rightMargin: 0
+        anchors.left: parent.left
+        anchors.leftMargin: 0
+        height: 64
+
+        Item {
+
+          Rectangle {
+            id: assetIcon
+            color: "gray"
+            width: 40
+            height: 40
+            anchors.left: parent.left
+            anchors.leftMargin: 0
+            anchors.top: parent.top
+            anchors.topMargin: 12
+            radius: 50
+          }
+
+          Rectangle {
+            id: transferIcon
+            anchors.topMargin: 25
+            anchors.top: parent.top
+            anchors.left: assetIcon.right
+            anchors.leftMargin: 22
+            height: 15
+            width: 15
+            color: "gray"
+
+            // TODO: implement transfer icon
+            // Image {
+            //     id: transferIcon
+            //     source: "..."
+            //     anchors.top: parent.top
+            //     anchors.left: assetIcon.right
+            //     anchors.leftMargin: Theme.smallPadding
+            // }
+          }
```

Comment:
@jrainville there are a bunch of icons in figma that weren't exportable for some reason. Put a square there for now, but maybe you can help?

Reply:
Everything should be exportable. Sometimes, you might have to use the layers panel (left tab view) and select the element there. Sometimes it's even the parent that you need to select, because the icon is made of multiple parts.
If you still have problems exporting it, you can ping me in Discord and we can quickly screenshare.